openc3/lib/openc3/packets/packet_config.rb

```diff
@@ -460,7 +461,30 @@ def process_current_packet(parser, keyword, params)
         rescue Exception => err
           raise parser.error(err)
         end
+
+      when 'RESPONSE'
+        usage = "#{keyword} <Target Name> <Packet Name>"
+        parser.verify_num_parameters(2, 2, usage)
+        @current_packet.response = [params[0].upcase, params[1].upcase]
+
+      when 'ERROR_RESPONSE'
+        usage = "#{keyword} <Target Name> <Packet Name>"
+        parser.verify_num_parameters(2, 2, usage)
+        @current_packet.error_response = [params[0].upcase, params[1].upcase]
```

Comment:
How will RESPONSE and ERROR_RESPONSE be used?

Reply:
This will be the standard way to build command/response patterns.  I'm using it with SNMP